src/templates/default.jsx

```diff
@@ -64,3 +48,29 @@ DefaultTemplate.propTypes = {
     }).isRequired,
   }).isRequired,
 };
+
+// can't use proptypes with special Gatsby Head component, so just ignore
+// eslint-disable-next-line react/prop-types
+export function Head({ data, ...rest }) {
+  return (
+    <SEO
+      // eslint-disable-next-line react/prop-types
+      title={data.markdownRemark.frontmatter.title}
+      keywords={[
+        `ICAT`,
```

Comment:
Looking at other files, it seems like single quote is used for strings instead of backticks, so for consistency's sake I think single quotes should be preferred here.